src/main/java/seedu/duke/commons/core/CommandFormat.java

```diff
@@ -34,6 +34,15 @@
     public static final String DONE_TASK_FORMAT =
             "done task [INDEX]";
 
+    public static final String EDIT_TASK_FORMAT =
+            "edit task [INDEX] -d {DAY_OF_THE_WEEK} -p {PRIORITY} -i {INFORMATION}";
+
+    public static final String EDIT_LESSON_FORMAT =
+            "edit lesson [INDEX] -d {DAY_OF_THE_WEEK} -s {START_TIME} -e {END_TIME} -l {MEETING_URL}";
```

Comment:
Since we are not using these we should remove them from here to avoid confusion for potential developers.

Reply:
Good point.